green_metric_tests/package.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "green_metric_tests",
+    "version": "1.0.0",
+    "description": "Integration tests for the Greencheck API, intended for use with the Green Metrics Tool from Green Code Berlin. (https://www.green-coding.io/projects/green-metrics-tool/)",
+    "main": "index.js",
+    "scripts": {
+      "test": "mocha '/*.spec.js'"

```

Comment:
When run in root, **/*.spec.js will try to read `procfs` on the default node container, which triggers the permissions error we saw